frontend/src/components/applications/EligibilityNotificationDialog.vue

```diff
@@ -0,0 +1,81 @@
+<template>
+  <AppDialog v-model="isDisplayed" title="Eligibility Notification" :is-loading="isLoading" persistent max-width="55%" @close="closeDialog">
+    <template #content>
+      <div class="confirm-dialog-text">
+        Thank you for your interest in joining the $10 a Day ChildCareBC Program. Based on the answers you have provided, your facility has not met the minimum eligibility criteria to participate at
+        this time. For more information about the $10 a Day ChildCareBC Program, please visit our
+        <a href="https://www2.gov.bc.ca/gov/content?id=F0D0FD2A00064CDDBF634CF83E4E2599" target="_blank">website</a>
```

Comment:
This is probably a mockup issue but should there be a period after the website link?

Reply:
it's updated